jupyter_scheduler/scheduler.py

```diff
@@ -473,6 +474,12 @@ def delete_job(self, job_id: str):
             if Status(job_record.status) == Status.IN_PROGRESS:
                 self.stop_job(job_id)
 
+            staging_paths = self.get_staging_paths(DescribeJob.from_orm(job_record))
```

Comment:
This looks unrelated, is this a fix for something else?

Reply:
This was raised by @dlq in a previous PR.
https://github.com/jupyter-server/jupyter-scheduler/pull/146#discussion_r998658523